roles/pythondev-site/handlers/main.yml

```diff
@@ -8,7 +8,7 @@
     - Restart pythondev
   when: systemd_enabled == true
 
-- name: Restart pythondev
+- name: Restart pythondev-site
```

Comment:
Looks like you may have missed other references to this name, such as on line 8 of this same file. Either that or we just have two redundant tasks here (which is probably the case)